### ISSUE TITLE
install.py scipy issue on rpi3b+

### ISSUE THREAD

**Issue by rekk2** [naztronaut/dancyPi-audio-reactive-led#63]
Hello.  I'm getting this error on a Raspberry Pi 3b+.  I did a manual build of scipy and still had issues.  This was the original error from install.py

/_lib/_uarray
      running build_clib
      customize UnixCCompiler
      customize UnixCCompiler using build_clib
      building 'mach' library
      error: library mach has Fortran sources but no Fortran compiler found
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for scipy
Failed to build scipy
ERROR: Could not build wheels for scipy, which is required to install pyproject.toml-based projects
================== Completed Installing Numpy, Scipy, PyAudio, PyQtgraph ==================

**Comment by rekk2** [comment]
the issue was with port audio.  I had to increase swap and manually build it, then install pyaudio 1.5.2 and then it worked

I also installed it on an raspberry pi 4 and had the same problem

**Comment by rox4H** [comment]
Hi there rekk2! Im having the same error as you and im using a RPi3b+, would you be more accurate on what you did to solve the issue? Thanks in advance

**Comment by rekk2** [comment]
I think i used this guide to increase the swap
https://arcanesciencelab.wordpress.com/2019/08/03/increasing-swapfile-space-on-the-raspberry-pi-4/
and then install version 1.5.2 of pyaudio because the others didnt work

**Comment by rox4H** [comment]
Thank you for answering so quickly! I have increased the swap but I can't see how to download version 1.5.2 of pyaudio, according to https://people.csail.mit.edu/hubert/pyaudio/ the newest version is 0.2.12. Could it be that you are referring to version 1.5.2 of scipy? Thanks again!

**Comment by rekk2** [comment]
Yes i forget exactly what i did.  Something only worked with version 1.5.2
